src/styles/blocks/card.css

```diff
@@ -0,0 +1,101 @@
+.card {
+  --primary-text-color: #060b35;
+  --secondary-text-color: #616070;
+
+  max-width: 200px;
+  min-height: 406px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  color: var(--primary-text-color);
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  object-fit: contain;
+  margin: 0 auto 40px;
+
+}
+
+.card__description {
+  flex-grow: 1;
+  margin-bottom: 16px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 1.5;
+  margin-bottom: 4px;
+}
```

Comment:
It would be better if you correct the order of writing styles using the example from this site
https://codeguide.academy/html-css.html#css-order

Reply:
Done